strapi/strapi
# Upgrade Tool

### What does it do?

Adds the upgrade tool

### Why is it needed?

To facilitate upgrading Strapi projects by automatically updating version numbers, installing packages, and running codemods for breaking changes.

### How to test it?

Within a Strapi project, run the upgrade tool command (located at packages/utils/upgrade/bin/upgrade.js )

So from examples/getstarted you would run: `../../packages/utils/upgrade/bin/upgrade.js --help` to see the list of commands

Note: The actual code transforms will be added throughout the development of v5 (for example, [this](https://github.com/strapi/strapi/pull/19155)), so at the moment running this tool doesn't cause any effects. So to really test it you will need to add some codemods to the resources dir.

### Related issue(s)/PR(s)
